lua/oil/fs.lua

```diff
@@ -146,7 +147,6 @@ end
 ---@param dir string
 ---@param mode? integer
 M.mkdirp = function(dir, mode)
-  mode = mode or 493
   local mod = ""
```

Comment:
According to the docs, `uv.fs_mkdir` expects a non-nil integer as the second argument. Did you intend to replace this with the value in the config file?

Reply:
Yes, the idea is to use the config value, and now we have a default one, so it's always available

Reply:
Then you'll need to update the function signature to make mode non-optional, and update the call here to pass the mode 
https://github.com/stevearc/oil.nvim/blob/c12fad2d225d8f81fadd48521d253607fe25465c/lua/oil/adapters/ssh.lua#L371